website/docusaurus.config.ts

```diff
@@ -31,6 +31,10 @@ const config: Config = {
     locales: ['en'],
   },
 
+  markdown: {
```

Comment:
Consider moving this commit to before Mermaid diagrams are actually introduced.

Reply:
It is before introducing them to the website, the previous conversion is still happening in the docs folder.